### PR TITLE
cp-196: Add Mobile User Profile

### PR DESCRIPTION
**Pull request by iamAlinaaa**
1. Added user profile info (username from authenticated user and placeholder image for now) to home screen.
2. Renamed Home component to Dashboard and in screen files too.
3. Sucessfully added User profile info
Снимок экрана 2023-09-06 163704

Снимок экрана 2023-09-08 124208

### CODE REVIEW

File: mobile/src/screens/dashboard/dashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const Dashboard: React.FC = () => {
+  const { userName } = useAppSelector(({ auth }) => {
+    return {
+      userName: (auth.authenticatedUser as UserAuthResponseDto).fullName,
+    };
+  });
+
+  return (
+    <LinearGradient>
+      <View style={styles.profileContainer}>
```
Comment (danilittaa) [inline mobile/src/screens/dashboard/dashboard.tsx:19]:
```suggestion
    <LinearGradient>
      <View style={styles.container}>
```

File: mobile/src/libs/enums/navigation/main-screen-name.enum.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const MainScreenName = {
-  HOME: 'Home',
+  DASHBOARD: 'Home',
```
Comment (Nanaier) [inline mobile/src/libs/enums/navigation/main-screen-name.enum.ts]: I think we should be naming our keys in the enum the same way we call our values

Comment (iamAlinaaa) [reply mobile/src/libs/enums/navigation/main-screen-name.enum.ts:2]: I had task to rename Home to Dashboard, but leave only title as Home, so not to be confused later i renamed all , but left title

**Review by OlenaYatseniuk — DISMISSED** [review]

**Review by dmytro-kucherenko — COMMENTED** [review]
Looks good, I'll wait until it gets out of draft)

**Review by BrianSammit — DISMISSED** [review]

File: mobile/src/screens/dashboard/dashboard.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+import userImagePlaceholder from '#assets/img/user-placeholder.png';
+import { Image, LinearGradient, Text, View } from '#libs/components/components';
+import { useAppSelector } from '#libs/hooks/hooks';
+import { type UserAuthResponseDto } from '#packages/users/users';
+
+import { styles } from './styles';
+
+const Dashboard: React.FC = () => {
+  const { userName } = useAppSelector(({ auth }) => {
```
Comment (fshabanov) [inline mobile/src/screens/dashboard/dashboard.tsx]:
```suggestion
  const { fullName } = useAppSelector(({ auth }) => {
```

**Review by fshabanov — CHANGES_REQUESTED** [review]

**Review by dmytro-kucherenko — DISMISSED** [review]

**Review by OlesiaTr — DISMISSED** [review]

File: mobile/src/assets/img/user-placeholder.png
Comment (liza-veis) [inline mobile/src/assets/img/user-placeholder.png:1]: let's use the same avatar as on the web

File: mobile/src/screens/dashboard/dashboard.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import { type UserAuthResponseDto } from '#packages/users/users';
+
+import { styles } from './styles';
+
+const Dashboard: React.FC = () => {
+  const { fullName } = useAppSelector(({ auth }) => {
+    return {
+      fullName: (auth.authenticatedUser as UserAuthResponseDto).fullName,
+    };
+  });
```
Comment (liza-veis) [inline mobile/src/screens/dashboard/dashboard.tsx]:
```suggestion
  const authenticatedUser = useAppSelector(({ auth }) => {
    return {
      authenticatedUser: auth.authenticatedUser as UserAuthResponseDto,
    };
  });
```

**Review by liza-veis — CHANGES_REQUESTED** [review]

**Review by liza-veis — APPROVED** [review]

**Review by v0ldemar01 — APPROVED** [review]

**Pull request merged**